### PR TITLE
update f_search_facts example

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: modules/foreman_search_facts.py [hunk 1]
```diff
@@ -70,7 +70,7 @@
     search: name = http_proxy
   register: result
 - debug:
-    var: result.response[0].value
+    var: result.resources[0].value
```
Comment (mdellweg) [inline modules/foreman_search_facts.py]: Has it always been resources? Or did this change with a foreman version?

Comment (evgeni) [reply modules/foreman_search_facts.py]: it was always resources, as it's in the module code ;)

Comment (mdellweg) [reply modules/foreman_search_facts.py]: Lucky us. Someone is looking at this stuff from time to time. ;)

Comment (evgeni) [reply modules/foreman_search_facts.py]: I can neither confirm nor deny that I was staring at my `apypie` branch for 5 minutes, wondering why it did not return me any results ;)

**Comment by sean797** [comment]
Thanks @evgeni, this was changed in the original PR, but I'm human and forgot to update this.

**Pull request merged**